src/libstore/unix/build/local-derivation-goal.cc

```diff
@@ -66,6 +66,14 @@
 
 namespace nix {
 
+namespace {
+/**
+ * The system for which Nix is compiled.
+ */
+[[gnu::unused]]
+constexpr const std::string_view nativeSystem = SYSTEM;
+}
+
```

Comment:
This seems to be unrelated. I think we can remove it for now, but I wonder what's it for?

```suggestion
```

Reply:
Yes. A cherry-pick mess up...